iree/samples/hal/CMakeLists.txt

```diff
@@ -12,6 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+add_custom_command(
+  OUTPUT simple_compute_test_module.emod
+  COMMAND iree-translate
+          --mlir-to-iree-module "${CMAKE_CURRENT_SOURCE_DIR}/simple_compute_test.mlir"
+          -o simple_compute_test_module.emod
+  # TODO(marbre): Get iree_translate target working.
+  #DEPENDS iree_translate
+)
+
+add_custom_command(
```

Comment:
Yes, we would prefer to wrap these in a `iree_bytecode_module` CMake function to reduce code duplication and make synchronizing between Bazel and CMake simpler.

For now, this is a step in the right direction and we can refactor later. Getting more parts of the project building with CMake is a great help :)